tests/test_geoparquet.py

```diff
@@ -0,0 +1,10 @@
+import os
+import geopandas as gpd
+from isamples_export_client.geoparquet_utilities import write_geoparquet_from_json_lines
+
+
+def test_geoparquet():
+    dest_file = write_geoparquet_from_json_lines("./test_data/isamples_export_2024_05_08_08_57_12.jsonl")
+    assert os.path.exists(dest_file)
+    df = gpd.read_parquet(dest_file)
+    assert df is not None
```

Comment:
@datadavev -- not sure what other kinds of checks we would want to make on the parquet DataFrame?  I poked around in the debugger and things are as I expect so I think it is legitimately working.